### PR TITLE
BLEN-105: Add RenderMan delegate samples option

### PR DESCRIPTION
**Pull request by VascoPi**
### PURPOSE
Add RenderMan delegate samples option.

### EFFECT OF CHANGE
Add RenderMan delegate Samples, Pixel Variance, and Timeout parameters.

### TECHNICAL STEPS
Improved _sync_render_settings for final and viewport render as well.
Added Prman properties hdprman_render.py.
- RenderSettings
  - samples
  - variance_threshold
  - timeout

Added Prman UI hdprman_render.py.
- HDUSD_RENDER_PT_hdprman_settings_final
- HDUSD_RENDER_PT_hdprman_settings_samples_final
- HDUSD_RENDER_PT_hdprman_settings_viewport
- HDUSD_RENDER_PT_hdprman_settings_samples_viewport

### NOTES FOR REVIEWERS
Added all parameters that are currently supported by USD.

### CODE REVIEW

**Review by ARozghon — APPROVED** [review]
manual and autotests completed without errors

File: src/hdusd/ui/hdprman_render.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+# limitations under the License.
+#********************************************************************
+from . import HdUSD_Panel
+
+
+#
+# FINAL RENDER SETTINGS
+#
+class HDUSD_RENDER_PT_hdprman_settings_final(HdUSD_Panel):
+    bl_label = "Prman Settings"
```
Comment (bsavery) [inline src/hdusd/ui/hdprman_render.py:22]: Should be RenderMan Settings (note capitalization)

File: src/hdusd/ui/hdprman_render.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        col.prop(hdprman, "samples")
+        col.prop(hdprman, "variance_threshold")
+        col.prop(hdprman, "timeout")
+
+
+#
+# VIEWPORT RENDER SETTINGS
+#
+class HDUSD_RENDER_PT_hdprman_settings_viewport(HdUSD_Panel):
+    bl_label = "Prman Settings"
```
Comment (bsavery) [inline src/hdusd/ui/hdprman_render.py:55]: RenderMan Settings

**Review by bsavery — COMMENTED** [review]
Update labels

**Pull request merged**